content/recipes/repl.md

```diff
@@ -0,0 +1,102 @@
+### Read-Eval-Print-Loop (REPL)
+
+// TODO: brief description on this feature
+
+#### Usage
+
+The REPL functionality is enabled when you call `repl` over some root NestJS module like so:
+
+```typescript
+@@filename(main)
+import { repl } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  await repl(AppModule);
+}
+bootstrap();
+@@switch
+import { repl } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  await repl(AppModule);
+}
+bootstrap();
+```
```

Comment:
We should probably instruct users to create a separate `repl.ts` file (instead of overriding the `main.ts`).